integrations/pinecone/tests/conftest.py

```diff
@@ -38,6 +38,9 @@ def document_store(request):
         dimension=dimension,
     )
 
+    store.initialize_index()
+    store.initialize_async_index()
```

Comment:
I would expect the single methods to be tested to call these initialization methods themselves.
Correct? If so, we should remove these 2 lines here.